lib/sorbet-rails/model_plugins/active_record_assoc.rb

```diff
@@ -138,13 +138,18 @@ def populate_collection_assoc_getter_setter(assoc_module_rbi, assoc_name, reflec
       return_type: relation_class,
     )
     unless assoc_should_be_untyped?(reflection)
-      if reflection.klass.table_exists?
-        # Normally the id_type is an Integer, but it could be a String if using
-        # UUIDs.
-        id_type = type_for_column_def(reflection.klass.columns_hash[reflection.klass.primary_key]).to_s
-      else
-        id_type = "T.untyped"
-      end
+      id_column = reflection.klass.columns_hash[reflection.klass.primary_key]
+      return if id_column.nil?
```

Comment:
@donaldong it shouldn't return here as that halts the method execution. It should only skip the generation of the "_ids" method.
```suggestion
      unless assoc_should_be_untyped?(reflection) && (id_column = reflection.klass.columns_hash[reflection.klass.primary_key])
         ...
      end
```

Reply:
Ah, thanks!